mlir/include/mlir/Dialect/MIOpen/Tuning/ConvContext.h

```diff
@@ -137,6 +146,18 @@ populateDimVal(const ArrayAttr &layoutAttr, const ArrayAttr &dimAttr,
   }
 }
 
+static inline void
+populateDimVal(const ArrayAttr &layoutAttr, const ArrayRef<int64_t> &dim,
+               llvm::StringMap<std::pair<size_t, int64_t>> &dimIndexVal) {
```

Comment:
If we are going to use this function on top level conv related op only then I prefer to get rid of the function with the same name above.

Reply:
In the next iteration I'll eliminate the function above.

Reply:
> The function is still needed for C++ emitters.

Good to know, thanks.